### ISSUE TITLE
CI fails when changed files have parens in file path

### ISSUE THREAD

**Issue by bsmth** [mdn/content#23605]
__Update:__

We have agreed to ban parentheses from file names and renamed the offending paths:
- [ ] https://github.com/mdn/content/pull/20470

CI checks are being put in place to ban new files from being added with parentheses:
- [ ] https://github.com/mdn/yari/pull/8019

---

__Failing workflow:__
https://github.com/mdn/content/actions/runs/3908866554/jobs/6679383140

Changed files:

* files/en-us/glossary/descriptor_(css)/index.md
* files/en-us/glossary/round_trip_time_(rtt)/index.md
* files/en-us/glossary/xhr_(xmlhttprequest)/index.md

Refers to line 5 in the script, which looks like:

https://github.com/mdn/content/blob/ebd19efc47d60ad38287f87f05ba624efe56ae2a/.github/workflows/pr-test.yml#L90-L95

i.e.:

```bash
yarn build files/en-us/glossary/descriptor_(css)/index.md
```

Where the following should work:

```
yarn build "files/en-us/glossary/descriptor_(css)/index.md"
```

Maybe we should try wrapping these file paths in quotes

CCing @yin1999 @schalkneethling

**Comment by yin1999** [comment]
I'll have a look on this

**Comment by yin1999** [comment]
> I'll have a look on this

Try to fix this in #23608

**Comment by OnkarRuikar** [comment]
- The PR https://github.com/mdn/content/pull/20470 is supposed to fix these file names. But is on hold for a long time, we need conclude the community discussion. May be bring up the issue in next meeting?

***

Command syntax, mainly shell expansions, are a little bit different for each shell -- newer shells like Z shell and fish are more forgiving than Bash. So scripts/commands that work in these shell may not work exactly the same on bash. If you are using Apple products then most likely you are using Z shell and not Bash.
Our GitHub workflows use `ubuntu-22.04` (need to check the shell and version). This is important because, I think the expansions are not happening the same as they happen in our local environment.

**Comment by estelle** [comment]
It looks like no files with a `(` can be edited right now. See https://github.com/mdn/content/pull/23620. I updated via the Github UI, and it's failing due to the parenthesis.

**Comment by OnkarRuikar** [comment]
> It looks like no files with a `(` can be edited right now. See #23620. I updated via the Github UI, and it's failing due to the parenthesis.

The change in PR is small and not affecting anything, so for now we can force merge it with failing workflow. And merge #20470 as soon as we can.

**Comment by yin1999** [comment]
Hi @bsmth, we may close this issue as we have merged #20470.

I'm also going to close the fixing PR.

**Comment by bsmth** [comment]
Thanks @yin1999 - I've updated the issue description with the related PRs.
